app.rb

```diff
@@ -1,9 +1,41 @@
 require 'sinatra/base'
+require './lib/message'
 
 class Chitter < Sinatra::Base
+  enable :sessions, :method_override
+
   get '/test' do
```

Comment:
You can get rid of this, the '/test' endpoint is created as a default but no need to keep it.